tests/conftest.py

```diff
@@ -0,0 +1,36 @@
+import os
+
+import pytest
+
+from flow360.file_path import flow360_dir
+from flow360.log import log, set_logging_file
+
+"""
+Before running all tests redirect all test logging to a temporary log file, turn off log rotation 
+due to multi-threaded rotation being unsupported at this time
+"""
+
+log_dir = flow360_dir + "logs"
+log_path = os.path.join(log_dir, "flow360_temp.log")
```

Comment:
`log_dir = os.path.join(flow360_dir, "logs")`
or combine with next line.

We should clean up after tests. Would suggest to log to a temp file which will be deleted at the end of the test

Reply:
Fixed in dbe1d20, only logs from test_log.py are not temporary and remain in a separate small file that is cleaned for every test run.